Godeps/Godeps.json

```diff
@@ -624,6 +629,111 @@
 			"ImportPath": "github.com/coreos/go-semver/semver",
 			"Comment": "v0.2.0-5-g5e3acbb",
 			"Rev": "5e3acbb5668c4c3deb4842615c4098eb61fb6b1e"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/vendor/github.com/Sirupsen/logrus",
```

Comment:
Is the nested vendoring going to create multiple versions of logrus/other packages in netplugin?